src/elements/content-explorer/__tests__/Content.test.tsx

```diff
@@ -88,12 +93,18 @@ describe('Content Component', () => {
         const features = {
             contentExplorer: { metadataViewV2: true },
         };
+        const collection = {
+            percentLoaded: 100,
+            boxItem: {},
+            id: '0',
+            items: [{ id: 1 }],
+            name: 'name',
+        };
 
         test('does not render MetadataBasedItemList when contentExplorer.metadataViewV2 is enabled', () => {
-            const collection = { boxItem: {}, id: '0', items: [{ id: 1 }], name: 'name' };
```

Comment:
does `Content` or any of the child components modify the original collection? if so it would cause instability in the tests to reuse the same collection object

Reply:
does it matter in that test suite? that collection is only used for the metadata views and in that "does not render test" is only to ensure that the v1 view doesnt render, majority of the time its for the v2 tests